src/cli/node/json_rpc_client.rs

```diff
@@ -563,18 +563,6 @@ impl fmt::Display for DataRequestTransactionInfo {
                     num_reveals,
                 )?;
             }
-            writeln!(
-                f,
-                "Commit rounds: {}/{}",
```

Comment:
Instead of removing this you can replace `{}/{}` with `{}`, and print `current_commit_round` but no `extra_commit_rounds`.